flexget/plugins/search_sceneaccess.py

```diff
@@ -10,11 +10,11 @@
 from flexget.event import event
 from flexget.utils.soup import get_soup
 from flexget.utils.search import torrent_availability, normalize_unicode, clean_title
-from flexget.utils.requests import Session
+from flexget.utils.requests import Session, TimedLimiter
 
 log = logging.getLogger('search_sceneaccess')
 
-session = Session()
+session = None
```

Comment:
Is it necessary to declare the variable in global scope? Isn't it only used in `search`?